src/api/instance-snapshots.tsx

```diff
@@ -43,7 +43,7 @@ export const deleteInstanceSnapshot = async (
   params.set("project", instance.project);
 
   return fetch(
-    `/1.0/instances/${encodeURIComponent(instance.name)}/snapshots/${encodeURIComponent(snapshot.name)}?${params.toString()}`,
+    `/1.0/instances/${encodeURIComponent(instance.name)}/snapshots/${encodeURIComponent(encodeURIComponent(snapshot.name))}?${params.toString()}`,
```

Comment:
This seems a bit odd, why do we have to encode it twice? Once should be enough. This might be a backend bug with double decoding on the API endpoint.